internal/cri/server/container_remove.go

```diff
@@ -59,6 +59,9 @@ func (c *criService) RemoveContainer(ctx context.Context, r *runtime.RemoveConta
 		// from the container store as well and return successfully.
 		log.G(ctx).WithError(err).Warn("get container info failed")
 		c.containerStore.Delete(ctrID)
+		if c.statsCollector != nil {
```

Comment:
over here I think.. https://github.com/containerd/containerd/blob/main/internal/cri/store/container/container.go#L197

Reply:
yeah but it's tied to the life of the container id... which are reusable ids

Reply:
i am missing something in `func (s *Store) Delete(id string)` i don't have access to either `criService` or `statsCollector` member of `criService`...